extensions/data-workspace/src/common/pathUtilsHelper.ts

```diff
@@ -49,42 +48,41 @@ export function sanitizeStringForFilename(s: string): string {
 /**
  * Returns true if the string is a valid filename
  * Logic is copied from src\vs\base\common\extpath.ts
- * @param name filename to check
+ * @param fileName filename to check (without file path)
  */
-export function isValidBasename(name?: string): boolean {
+export function isValidBasename(fileName?: string): boolean {
```

Comment:
Instead of having two functions doing the same kinds of checks it would be easier to have the error message one return undefined if it's a valid basename and then this can just call that and test for if the message is undefined.

Reply:
Completely makes sense. Updated the code appropriately.